openquake/hazardlib/gsim/kotha_2020.py

```diff
@@ -139,12 +145,37 @@ def get_distance_coefficients_2(kind, c3, c3_epsilon, C, imt, sctx):
     return c3_ + c3_epsilon * tau_c3
 
 
-def get_distance_term(kind, c3, c3_epsilon, C, ctx, rjb, imt):
+@get_distance_coefficients.add("regional")
+def get_distance_coefficients_3(kind, c3, delta_c3_epsilon, C, imt, sctx):
+    """
+    Return site-specific coefficient 'C3'. The function retirieves the 
+    value of delta_c3 and the standard error of delta_c3 from the 'att' 
+    geojson file depending on the location of site. This delta_c3 is 
+    added to the generic coefficient 'c3' from the GMPE. A delta_c3_epsilon
+    value of +/- 1.6 gives the 95% confidence interval for delta_c3.
+    """
+    s = []
+    [s.append(Point(lon, lat)) for lon, lat in zip(sctx.lon, sctx.lat)]
```

Comment:
The part of reading the input file should be moved to the __init__ function. In this way, you avoid reading the same file thousand of times.

Reply:
The geojson files have been instantiated in the **init** function now.